scripts/migrations/olids_2026/rewrite_staging.py

```
@@ -0,0 +1,385 @@
+        for col in rep.removed:
+            print(f"  remove : {col}")
+        for col in rep.cast:
+            print(f"  cast   : {col}")
+        for col in rep.additions:
+            print(f"  add    : {col}  (TODO placeholder emitted)")
+        for note in rep.needs_manual:
+            print(f"  MANUAL : {note}")
+
+    unhandled = sorted({c for r in reports for c in r.unchanged_cols})
+    if unhandled:
+        print(f"\nUnhandled columns (pass-through, may want review):")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)